app/parsers/dto_converter.py

```
@@ -0,0 +1,212 @@
+            if isinstance(item, list):
+                # 딕셔너리 리스트에서 정보 추출
+                mortgage_info = None
+                for data in item:
+                    if isinstance(data, dict):
+                        # 순위번호 찾기
+                        priority_num = None
+                        for key, value in data.items():
+                            if "순위번호" in str(key):
+                                try:
+                                    priority_num = int(re.search(r'\d+', str(value)).group()) if re.search(r'\d+', str(value)) else priority_counter
+                                except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)